src/botPage/view/TradeInfoPanel/TradeTable.js

```diff
@@ -47,17 +47,17 @@ export default class TradeTable extends Component {
             },
         };
         this.columns = [
-            { key: 'timestamp', width: 192, resizable: true, name: translate('Timestamp') },
-            { key: 'reference', width: 110, resizable: true, name: translate('Reference') },
+            { key: 'timestamp', width: 182, resizable: true, name: translate('Timestamp') },
+            { key: 'reference', width: 100, resizable: true, name: translate('Reference') },
```

Comment:
Why are we reducing width here?

Reply:
Table has fixed width. I enlraged 'Status', so I had to make something else thinner